internal/engine/experiment/tor/tor_test.go

```diff
@@ -258,8 +257,8 @@ func TestResultsCollectorMeasureSingleTargetGood(t *testing.T) {
 		new(model.Measurement),
 		model.NewPrinterCallbacks(log.Log),
 	)
-	rc.flexibleConnect = func(context.Context, keytarget) (oonitemplates.Results, error) {
-		return oonitemplates.Results{}, nil
+	rc.flexibleConnect = func(context.Context, keytarget) (*measurex.ArchivalMeasurement, *string) {
+		return &measurex.ArchivalMeasurement{}, nil
```

Comment:
Here I needed to adapt `flexibleConnect` (remember: it always returns a non-nil measurement)